### ISSUE TITLE
The name property of an element should not contain class names like 'Microsoft.*.*' or 'Windows.*.*' as these are not usually information.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#9066]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yAuto; #AI4D; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG4.2.1; #GH_.NETMAUI_Win32_Jul2022; #Object Information​; #A11yeDAD;

### Description

The name property of an element should not contain class names like 'Microsoft.*.*' or 'Windows.*.*' as these are not usually information.

### Steps to Reproduce

1. Download the Zip file "pipeline-ui-maui-windows-1.0.0.1.zip" [https://github.com/NPavie/pipeline-ui-maui/releases/download/prototyp-v1/pipeline-ui-maui-windows-1.0.0.1.zip](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNPavie%2Fpipeline-ui-maui%2Freleases%2Fdownload%2Fprototyp-v1%2Fpipeline-ui-maui-windows-1.0.0.1.zip&data=05%7C01%7Cv-karanmore%40microsoft.com%7C9ede9d1f3bd94832f69b08da70c0255f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637946267837269549%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oq2mgje0beuTFasZdyvgf4hN8xLX2dyB903t%2Byejnbs%3D&reserved=0). 
2. Extract the "pipeline-ui-maui-windows-1.0.0.1.zip" file.
3. open the extracted file and navigate till "pipeline-ui-maui.exe" and press enter.
4. the New Job screen will open.
5. Start Accessibility insight for Desktop and see results.
![The name property of an element should not contain class names like 'Microsoft ' or 'Windows ' as these are not usually information](https://user-images.githubusercontent.com/100186395/181736536-820db358-3080-4cf6-b8ee-b516e8e7b085.png)

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 11 Enterprise Version 21H2 OS build 22000.795

### Relevant log output

```shell
The name property of an element should be contain class names like 'Microsoft.*.*' or 'Windows.*.*' as these are not usually information.
```
